packages/cashier/src/Components/Form/crypto-transactions-history/crypto-transactions-cancel-modal.jsx

```diff
@@ -28,6 +29,7 @@ const CryptoTransactionsCancelModal = ({
                         text={localize('Yes')}
                         onClick={() => {
                             cancelCryptoTransaction(selected_crypto_transaction_id);
+                            onMount();
```

Comment:
@artsemkavaleu-deriv no need to call this here, we are already loading the transactions on cancel.